lib/ansible/plugins/inventory/cloudscale.py

```diff
@@ -62,10 +62,22 @@
 '''
 
 EXAMPLES = r'''
-# cloudscale_inventory.yml file in YAML format
-# Example command line: ansible-inventory --list -i cloudscale_inventory.yml
+# cloudscale.yml file in YAML format
```

Comment:
Do you mind also documenting in the plugin description or notes that the file must end with cloudscale.yml or cloudscale.yaml?